src/components/NewAppToolBar/NewAppToolBar.scss

```diff
@@ -0,0 +1,106 @@
+
+.new-app-tool-bar {
+    &-container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 100vw;
+        padding: 1.5rem;
+    }
+
+    &-arrow-right {
+        width: 1rem;
+        height: 1rem;
+        align-self: center;
+    }
+
+    &-main-account-logo {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        gap: 1rem;
+        margin-right: 1.438rem;
+
+        &-img {
+            width: 48px;
+            height: 48px;
+        }
+
+        &-text {
+            font-family: 'Source Sans Pro', sans-serif;
+            font-weight: 600;
+            line-height: 1.25rem;
+            font-size: 1.125rem;
+        }
+    }
+
+    &-new-app-logo {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        gap: 1rem;
+        margin-left: 1.438rem;
+
+        &-img {
+            width: 48px;
+            height: 48px;
+        }
+
+        &-text {
+            font-family: 'Source Sans Pro', sans-serif;
+            font-weight: 600;
+            line-height: 1.25rem;
+            font-size: 1.125rem;
+        }
+    }
+
+    &-status {
+        &-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            gap: 0.375rem;
+        }
+
+        &-icon {
+            width: 0.5rem;
+            height: 0.5rem;
+            filter: invert(54%) sepia(5%) saturate(427%) hue-rotate(201deg) brightness(94%) contrast(98%);
+        }
+
+        &-text {
+            font-family: 'Source Sans Pro', sans-serif;
+            font-weight: 600;
+            color: #6C6C75; 
+        }
+    }
+
+    &-button {
+        &-container {
+            display: flex;
+            gap: 1rem;
+        }
+
+
+        &-text {
+            font-family: 'Source Sans Pro', sans-serif;
+            font-size: 0.875rem;
+            line-height: 1rem;
+            font-weight: 600;
+        }
+
+        &-exit {
+            color: #2B3A4B;
```

Comment:
@danvb112 You can create a variable to store the hexadecimal value and call it in all places you needed. This would make easier to change the same color in many places later.